tabs/osd.js

```diff
@@ -340,10 +342,9 @@ OSD.updateDisplaySize = function() {
 OSD.msp = {
   encodeOther: function() {
     var result = [-1, OSD.data.video_system];
-    if (semver.gte(CONFIG.apiVersion, "1.21.0")) {
+    if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
```

Comment:
So everybody that uses MSP (mobile tools, OSD, ...) will have to care about the versioning we are using for Betaflight?